helm/g2p-sandbox/Chart.yaml

```diff
@@ -7,5 +7,5 @@ appVersion: 0.0.0
 
 dependencies:
 - name: ph-ee-engine
-  repository: https://fynarfin.io/images/ph-ee-engine-0.0.0-SNAPSHOT
-  version: 0.0.0-SNAPSHOT
+  repository: https://fynarfin.io/images/ph-ee-engine-1.8.0-SNAPSHOT
+  version: 1.8.0-SNAPSHOT
```

Comment:
this will not work with circle CI pipeline I think

Reply:
reverted back to v0.0.0